notebooks/interfield_differences.ipynb

```diff
@@ -0,0 +1,1245 @@
+{
```

Comment:
<p>I also wonder if these rankings are, at least somewhat, a reflection of the "community" that the authors belong to or publish in. For example I know that Brendan Frey (senior author on "Deep learning of the tissue-regulated splicing code") publishes, or at least has published, in ML conferences like NeurIPS, ICML, etc. so maybe his work is more visible to the AI community. Conversely it could be the case that the "high" genetics-AI work comes from authors that are in medical schools or less AI/CS-oriented departments, so it's more visible to the genetics community than the AI community.</p><p> </p><p>Anyway, just a (super anecdotal) thought - not sure exactly how you would test that empirically.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/indices/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='92'/>

Reply:
<p>I agree that that should be some (or maybe most) of the signal that differentiates these papers. I'm currently working on figuring out how to differentiate luck and the field-specific signals like author visibility</p>